languageserver/conversion/conversion.go

```diff
@@ -127,6 +127,8 @@ func DeclarationToDocumentSymbol(declaration ast.Declaration) protocol.DocumentS
 			name = "init"
 		case common.DeclarationKindDestructor:
 			name = "destroy"
+		case common.DeclarationKindImport:
```

Comment:
That should fix the case when the program has an import declaration 👍 

I had a quick look and there are some more cases where the declaration does not have an identifier (`DeclarationIdentifier()` returns `nil`), e.g. pragmas.

Instead of handling specific cases in the fallback (when there is no identifier available), maybe we can instead handle this in general by using the `DeclarationKind.Keywords()`, it pretty much returns what we would like to see as a document symbol name. 

Could you please replace the whole switch case with just this (GitHub does not let me provide a suggestion for non-edited code):
```go
name = declarationKind.Keywords()
```

Reply:
nice!  didn't know this existed.  threw that in there as well as a filter that all symbols must have a non-empty name just in case declaration.Keywords() is missing symbols in the future.

Reply:
Perfect, thanks!